addOns/automation/src/main/java/org/zaproxy/addon/automation/AutomationEnvironment.java

```diff
@@ -170,6 +173,46 @@ public ContextWrapper parseContextData(
                                         "automation.error.context.badurl", value.toString()));
                     }
                     break;
+                case "includePaths":
+                    if (!(value instanceof ArrayList)) {
+                        progress.error(
+                                Constant.messages.getString(
+                                        "automation.error.context.badincludelist", value));
+                    } else {
+                        includeRegexes = (ArrayList<?>) value;
+                        for (Object regex : includeRegexes) {
+                            try {
+                                Pattern.compile(regex.toString());
+                            } catch (PatternSyntaxException e) {
+                                progress.error(
+                                        Constant.messages.getString(
+                                                "automation.error.context.badregex",
+                                                regex.toString(),
+                                                e.getMessage()));
+                            }
+                        }
+                    }
```

Comment:
This could be extracted into a method (e.g. pass the `value` and the message key and return the list).